torchci/lib/bot/ciflowPushTrigger.ts

```diff
@@ -136,7 +141,16 @@ async function handleLabelEvent(
   if (!isCIFlowLabel(label)) {
     return;
   }
-  const config: any = await tracker.loadConfig(context);
+  if (repoKey(context) == "pytorch/pytorch-canary"){
+    const config: any =  await Octokit.config.get({
+      owner: `${context.repo().owner}`,
+      repo: `${context.repo().repo}`,
+      path: ".github/pytorch-probot.yml",
+      branch: `${context.payload.ref}`,
+    });
+  }else{
```

Comment:
What effect does this have? (i.e. how is this config different than the non-canary version?)